src/python/pants/backend/native/tasks/cpp_compile.py

```diff
@@ -32,27 +32,16 @@ def subsystem_dependencies(cls):
     )
 
   @memoized_property
-  def _toolchain(self):
+  def _native_toolchain(self):
     return NativeToolchain.scoped_instance(self)
 
   def get_compile_settings(self):
     return CppCompileSettings.scoped_instance(self)
 
+  @memoized_property
+  def _cpp_toolchain(self):
+    llvm_cpp_toolchain = self._request_single(LLVMCppToolchain, self._native_toolchain)
```

Comment:
Another instance of regressing? to LLVM lock in here in a presumably generic CppCompile class.

Reply:
Yep! This was fixed in 68435790b9f! Thanks for describing the regressions!